client/src/containers/AddTrip/index.js

```diff
@@ -40,24 +59,41 @@ class AddTrip extends Component {
 
   onEditorStateChange = (editorState) => {
     const newFormdata = { ...this.state.formdata };
+    const newHideError = {
+      ...this.state.hideError,
+      description: true,
+    };
     const contentState = editorState.getCurrentContent();
     const rawState = convertToRaw(contentState);
 
     newFormdata.description = JSON.stringify(rawState);
 
-    this.setState({ editorState, formdata: newFormdata });
+    this.setState({ editorState, formdata: newFormdata, hideError: newHideError });
+  }
+
+  getErrorClass(fieldName) {
+    return this.formFieldHasError(fieldName) ? 'field-error' : '';
```

Comment:
here is better to put `' field-error'` and to remove the space in the place where the class inserted to the string...

Reply:
Here it is not better, but in StartsRating it worked out pretty well! :D